src/plugin.ts

```diff
@@ -1,47 +1,83 @@
 import { Octokit } from "@octokit/rest";
-import { createClient } from "@supabase/supabase-js";
-import { createAdapters } from "./adapters";
-import { Env, PluginInputs } from "./types";
+import { PluginInputs } from "./types";
 import { Context } from "./types";
+import { askQuestion } from "./handlers/ask-llm";
+import { addCommentToIssue } from "./handlers/add-comment";
+import { LogLevel, LogReturn, Logs } from "@ubiquity-dao/ubiquibot-logger";
```

Comment:
Does this package still work? I thought we deleted it and rebranded to something like

@ubiquity-os/ubiquity-os-logger